pom.xml

```diff
@@ -25,7 +25,7 @@
         <hibernate.version>5.2.12.Final</hibernate.version>
         <flywaydb.version>5.0.5</flywaydb.version>
         <jackson.version>2.9.5</jackson.version>
-        <lombok.version>1.16.20</lombok.version>
+        <lombok.version>1.18.6</lombok.version>
```

Comment:
Since most of the changes are related to updated java version, then please also change java.version property.